src/store/index.ts

```diff
@@ -317,7 +320,8 @@ const useTaxesStore = defineStore({
       return {
         year: this.grossIncome.year - this.irsPay.year - this.ssPay.year,
         month: monthIncome,
-        day: monthIncome / MONTH_BUSINESS_DAYS,
+        day: (this.grossIncome.year - this.irsPay.year - this.ssPay.year) / 
```

Comment:
same as before, make this year value a variable before the return statement, to avoid duplication